app/uk/gov/hmrc/plasticpackagingtax/registration/services/GRSRedirections.scala

```diff
@@ -85,4 +87,20 @@ trait GRSRedirections extends I18nSupport {
       grsUrl
     )
 
+  def getRegisteredSocietyRedirectUrl(grsUrl: String, callbackUrl: String)(implicit
+    request: Request[_],
+    headerCarrier: HeaderCarrier
+  ): Future[String] =
+    registeredSocietyGrsConnector.createJourney(
+      IncorpEntityGrsCreateRequest(callbackUrl,
```

Comment:
Could this be pulled out into a private function and shared with the UK company connector?

Reply:
Done